packages/cfd/src/Containers/derivx-trade-modal.tsx

```diff
@@ -206,14 +206,14 @@ const DerivXTradeModal = ({
                                 const account_type = getCFDAccountKey({
                                     market_type: mt5_trade_account.market_type,
                                     sub_account_type: mt5_trade_account.sub_account_type,
-                                    platform: CFD_PLATFORMS.DMT5,
+                                    platform: CFD_PLATFORMS.MT5,
                                     shortcode: mt5_trade_account.landing_company_short,
                                 });
                                 onPasswordManager(
                                     mt5_trade_account?.login,
                                     getTitle(mt5_trade_account.market_type, is_eu_user),
                                     mt5_trade_account.account_type,
-                                    account_type,
+                                    account_type || '',
```

Comment:
@hamza-deriv how account_type could be `' '`?

Reply:
The thing with the account_type is that it is used as same variable for different properties like market_type and type are used in the same variable and i think we need to refactor in all occurences and it will requires new methods separately for market_type and type and it is a huge change. After that it can be typed correctly otherwise it will break the whole cfd package